mappings/mappings.php

```diff
@@ -26,9 +26,10 @@
             'baidu'                               => 'baiduspider',
```

Comment:
Is it possible to keep these keys alphabetized so that changes to this file are easier to grok when there are a lot of differences?

Reply:
changed